docs/micro-rdk/movement-sensor/accel-adxl345.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Configure an ADXL345 Accelerometer"
+linkTitle: "accel-adxl345"
+weight: 20
+type: "docs"
+description: "Configure an ADXL345 digital accelerometer."
+images: ["/icons/components/imu.svg"]
+# SMEs: Rand, Kim Mishra
+---
+
+The `accel-adxl345` movement sensor model supports the Analog Devices [ADXL345 digital accelerometer](https://www.analog.com/en/products/adxl345.html).
+This three axis accelerometer supplies linear acceleration data, supporting the `LinearAcceleration` method.
+
+If you are using a [Viam Rover](https://docs.viam.com/try-viam/), this is the accelerometer on it.
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and click **Create component**.
+Select the `movement-sensor` type, then select the `accel-adxl345` model.
+Enter a name for your movement sensor and click **Create**.
+
+{{< imgproc src="/components/movement-sensor/adxl345-builder.png" alt="Creation of an `accel-adxl345` movement sensor in the Viam app config builder." resize="600x" >}}
+
+Copy and paste the following attribute template into your movement sensor's **Attributes** box.
+Then remove and fill in the attributes as applicable to your movement sensor, according to the table below.
+
+{{< tabs >}}
+{{% tab name="Attributes template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "board": "<your-board-name>",
+  "i2c_bus": "<your-i2c-bus-name-on-board>",
+  "use_alternate_i2c_address": <boolean>,
```

Comment:
a reminder, for micro-RDK this is "use_alt_i2c_address"

Reply:
Ah yes did not catch that here! Thank you! Changed